### ISSUE TITLE
chroots stop booting up

### ISSUE THREAD

**Issue by gitrubus** [dnschneid/crouton#4761]
```
Please paste the output of the following command here: name: jammy
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/jammy...
crouton: version 1-20220111101058~master:a9eafaee
release: impish
architecture: amd64
xmethod: xorg
targets: unity-desktop,chrome,chrome-dev,chrome-beta,chromium,core,audio,kodi,x11,xorg,touch,keyboard,gtk-extra,core
host: version 14588.123.0 (Official Build) stable-channel hatch 
kernel: Linux localhost 4.19.234-12910-g254ead18a8fe #1 SMP PREEMPT Wed May 18 19:35:34 PDT 2022 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
Unmounting /mnt/stateful_partition/crouton/chroots/jammy...
name: kali-rolling
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/kali-rolling...
crouton: version 1-20220111101058~master:a9eafaee
release: kali-rolling
architecture: amd64
xmethod: xorg
targets: xfce,chrome,kodi,chromium,lxde,cli-extra,gtk-extra,keyboard,x11,xorg
host: version 14588.123.0 (Official Build) stable-channel hatch 
kernel: Linux localhost 4.19.234-12910-g254ead18a8fe #1 SMP PREEMPT Wed May 18 19:35:34 PDT 2022 x86_64 GNU/Linux
freon: yes
Unmounting /mnt/stateful_partition/crouton/chroots/kali-rolling...
```

#### Please describe your issue: okay I was trying to install kali linux and the this came up 

- [ ] Entering /mnt/stateful_partition/crouton/chroots/kali-rolling...
/usr/bin/startxfce4: Starting X server

_XSERVTransmkdir: Owner of /tmp/.X11-unix should be set to root

X.Org X Server 1.21.1.3
X Protocol Version 11, Revision 0
Current Operating System: Linux localhost 4.19.234-12910-g254ead18a8fe #1 SMP PREEMPT Wed May 18 19:35:34 PDT 2022 x86_64
Kernel command line: cros_secure console= loglevel=7 init=/sbin/init cros_secure drm.trace=0x106 root=/dev/dm-0 rootwait ro dm_verity.error_behavior=3 dm_verity.max_bios=-1 dm_verity.dev_wait=1 dm="1 vroot none ro 1,0 5324800 verity payload=PARTUUID=e36299a5-45e7-4e9a-82c3-430de5a57137/PARTNROFF=1 hashtree=PARTUUID=e36299a5-45e7-4e9a-82c3-430de5a57137/PARTNROFF=1 hashstart=5324800 alg=sha256 root_hexdigest=56cd252eba7780da0f06371d3ff5a1a3410b2edcd77c5976f1c605770c50e116 salt=46a973af35e545e6887f6a3d03cf3844a13b0070017302ade35b8c3d6fe80bee" noinitrd vt.global_cursor_default=0 kern_guid=e36299a5-45e7-4e9a-82c3-430de5a57137 add_efi_memmap boot=local noresume noswap i915.modeset=1 ramoops.ecc=1 gsmi.s0ix_logging_enable=1 intel_pmc_core.warn_on_s0ix_failures=1  
xorg-server 2:21.1.3-2+b1 (https://www.debian.org/support) 
Current version of pixman: 0.40.0
        Before reporting problems, check http://wiki.x.org
        to make sure that you have the latest version.
Markers: (--) probed, (**) from config file, (==) default setting,
        (++) from command line, (!!) notice, (II) informational,
        (WW) warning, (EE) error, (NI) not implemented, (??) unknown.
(++) Log file: "/tmp/Xorg.crouton.1.log", Time: Sun Jun  5 18:44:42 2022
(==) Using system config directory "/usr/share/X11/xorg.conf.d"
error: Kernel is too old for Iris. Consider upgrading to kernel v4.16.

(EE) 
Fatal server error:
(EE) AddScreen/ScreenInit failed for driver 0
(EE) 
(EE) 
Please consult the The X.Org Foundation support 
         at http://wiki.x.org
 for help. 
(EE) Please also check the log file at "/tmp/Xorg.crouton.1.log" for additional information.
(EE) 
(EE) Server terminated with error (1). Closing log file.
/usr/bin/xinit: giving up
/usr/bin/xinit: unable to connect to X server: Connection refused
/usr/bin/xinit: server error
Unmounting /mnt/stateful_partition/crouton/chroots/kali-rolling...

**Comment by TomTravis** [comment]
yes Xorg fails on Kali-rolling with crouton Chromebook it runs fine on my server chroot
likely the crouton tasks are not starting or have been started, it a hack
Xiwi runs but is damaged also
(EE)
Fatal server error:
(EE) AddScreen/ScreenInit failed for driver 0
(EE)
(EE)
this is normal for the server it cannot connect to a display
The crouton server has is not starting via xinit, see /usr/local/bin/xinit for the process flow

**Comment by gitrubus** [comment]
I don't use extentions

**Comment by gitrubus** [comment]
how do you fix it????

**Comment by gitrubus** [comment]
is there a real way how to fix it?

**Comment by gitrubus** [comment]
okay im seeing the problem or its just the matence or im crazy but why does it say ubuntu/prepare on kali and debian?
